### PR TITLE
Revert "Changing golang version to 1.16.11 in go-algorand"

### PR DESCRIPTION
**Pull request by algojack**
Reverts algorand/go-algorand#2825 to back out 1.16 golang update

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Install specific golang
         uses: actions/setup-go@v2
         with:
-          go-version: '1.16.11'
+          go-version: '1.16.6'
```
Comment (onetechnical) [inline .github/workflows/reviewdog.yml:47]: Is this correct?

**Review by onetechnical — COMMENTED** [review]
Just a question before I approve.

**Review by onetechnical — APPROVED** [review]
I guess that's just how it was.

**Comment by onetechnical** [comment]
Can you re-merge master? We have a separate change that should help the CI tests pass.

**Review by egieseke — APPROVED** [review]

**Review by onetechnical — APPROVED** [review]

**Pull request merged**